byte NO_PATH = 0
byte FAR_FROM_PATH = 1
byte FOLLOWING = 2
byte GOAL = 3

Header header
float32 distance_remains
float32 angle_remains
int32 status

